pkg/test/template/util.go

```diff
@@ -0,0 +1,25 @@
+package template
+
+import (
+	"fmt"
+
+	templatev1 "github.com/openshift/api/template/v1"
+	errs "github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// DecodeTemplate decodes bytes to Template object
+func DecodeTemplate(decoder runtime.Decoder, tmplContent []byte) (*templatev1.Template, error) {
+	if decoder == nil {
+		return nil, fmt.Errorf("decoder cannot be nil")
+	}
+	obj, _, err := decoder.Decode(tmplContent, nil, nil)
+	if err != nil {
+		return nil, errs.Wrapf(err, "unable to decode template")
+	}
+	tmpl, ok := obj.(*templatev1.Template)
+	if !ok {
+		return nil, fmt.Errorf("unable to convert object type %T to Template, must be a v1.Template", obj)
+	}
```

Comment:
actually, you can do it in a simpler way, without having to convert the returned object into a `*templatev1.Template`:

```suggestion
tmplObj := &templatev1.Template{}
_, _, err = g.decoder.Decode(tmpl, nil, tmplObj)
if err != nil {
	return nil, errs.Wrapf(err, "unable to decode template")
}
```

Reply:
@xcoulon with this simpler way I don't thing this wrapper function required in `common/test`.  We can directly call `decoder.Decode(tmplContent, nil, tmplObj)` in our test_code wherever required .. WDYS ??

Reply:
right, the function body is pretty straightforward, indeed, and this func here adds no real benefit, so we can probably close this PR.

Reply:
Thanks for simpler way suggestion :-)  Closing this PR.